tinygrad/codegen/linearizer.py

```diff
@@ -64,6 +64,10 @@ def get_reduce_acc(self, op, dtype:DType):
     AndNode: lambda self,ops,ctx:
       functools.reduce(lambda a,b: ctx.uop_alu_idx(a, b, ops, ctx, BinaryOps.MUL, dtype=dtypes.bool), self.nodes[1:], self.nodes[0].render(ops,ctx)) }
 
+  def get_uop_dtype(self, dtype: DType, amt=1):
+    if isinstance(dtype, ImageDType): dtype = dtypes.float
```

Comment:
also should this be `dtype.base` and not hard coded to float here?

Reply:
oh cool we finally have that